.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+## Description of changes  
+Provide a description of what this PR does. What bug does it fix, or what feature does it add? Do you expect that this PR will change answers, and if so, under what circumstances?
+
+## Tests conducted
+What testing has been conducted on the PR thus far? Describe the nature of any scientific or technical tests, including relevant details about the configuration(s) (e.g., cold versus warm start, number of cycles, forecast length, whether data assimilation was performed, etc). What platform(s) were used for testing?
+
+## Dependencies
+If submodule PRs are required, please link them below. For example:
+- hafs-community/ufs-weather-model/pull/<pr_number>
+- hafs-community/GSI/pull/<pr_number>
+- hafs-community/ccpp-physics/pull/<pr_number>
+
+## Issues addressed (optional)
+If this PR addresses one or more issues, please provide link(s) to the issue(s) here.
+
```

Comment:
@evankalina it would probably be better to move this Issues addressed section right after the Description section.

Reply:
Good idea. I moved the section as suggested.